FilterRules/ageatdeath.py

```diff
@@ -97,6 +97,6 @@ def prepare(self, db, user):
                     elif leg == "greater than" and age[0] > max_age:
                         self.ref_list.add(person_h)
 
-    def apply(self, db, person):
+    def apply_to_one(self, db, person):
         """Check if the filter applies to the person."""
         return person.handle in self.ref_list
```

Comment:
@dsblank this is the minimal change to allow the rule to work again.
Is there any advantage to be gained by converting `self.ref_list` to `self.selected_handles`, other than consistency?

Reply:
Yes, that is the minimal change. I think the name is fine... there are all kinds of names used throughout gramps-addons and gramps.

Reply:
Oh, I see. You are correct, `selected_handles` is a special name. Can you update all of these?

Reply:
I'll look a bit later today

Reply:
> Oh, I see. You are correct, `selected_handles` is a special name. Can you update all of these?

If a rule matches only a single object (e.g. `IsActivePerson`) does it still make sense to add the handle to a property called `selected_handles`? I feel like it should allow the `Optimizer` class to kick in and do a set operation rather than calling `apply_to_one` multiple times.
My doubt comes from the core rule `HasGrampsId` which can also only match at most one object and does not use `selected_handles`. It could call `db._get_raw_person_from_id_data` and populate `selected_handles`